tests/how_to_test_systems.rs

```diff
@@ -32,14 +32,12 @@ fn despawn_dead_enemies(
     }
 }
 
-fn hurt_enemies(mut enemies: Query<&mut Enemy>) {
-    for mut enemy in &mut enemies {
-        enemy.hit_points -= 1;
-    }
+fn hurt_enemies(mut enemies: Query<&mut Enemy>){
+    enemies.iter_mut().for_each(|mut enemy| enemy.hit_points -= 1);
 }
 
-fn spawn_enemy(mut commands: Commands, keyboard_input: Res<ButtonInput<KeyCode>>) {
-    if keyboard_input.just_pressed(KeyCode::Space) {
+fn spawn_enemy(mut commands: Commands, keyboard_input: Res<Input<KeyCode>>) {
```

Comment:
I don't think this change compiles.

Reply:
I was of the idea that the tweak being to the generic type parameter would not cause any compile errors

Reply:
Yeah, this was renamed from `Input` to `ButtonInput` in Bevy 0.13 or so :)